### PR TITLE
line edits

### PR DESCRIPTION
**Pull request by sascala**
relates to: https://mesosphere.atlassian.net/browse/DOCS-1347

### CODE REVIEW

File: docs/docs/deployments.md [hunk 1]
```diff
@@ -5,118 +5,111 @@ title: Application Deployments
...
-Deployments are not instantly available - they take time. 
-A deployment is active in Marathon until the deployment is finished successfully.
+Deployments take time and are not instantly available. A deployment is considered active in Marathon until it has finished successfully.
 
-Multiple deployments can be performed at the same time, as long as one application is changed only by one deployment.
-If a deployment is requested, that tries to change an application that is already being changed by another active deployment, 
-the new deployment request will be rejected.
+You can perform multiple deployments at the same time as long as one application is changed only by one deployment.
+If you request a deployment that tries to change an application that is already being changed by another active deployment, 
+the new deployment request Marathon will reject the new deployment.
```
Comment (jdef) [inline docs/docs/deployments.md]: this looks fudged

File: docs/docs/deployments.md [hunk 2]
```diff
@@ -5,118 +5,111 @@ title: Application Deployments
...
+These capacity constraints are added together if there are dependencies. In our example, we defined 0.6 for _db_ and 0.8 for _app_. 
+This means that when an update is performed there will be 12 instances of _db_ (6 old and 6 new) and 32 instances of _app_ (16 old and 16 new) running side by side.
 
 ## Force a Deployment
 
 An application can be changed by only one deployment at a time.
 Other changes to the application must wait until the first deployment has finished.
-It is possible to break this rule by running a deployment with the force flag.
-The REST interface allows the force flag for all state-changing operations.
+You can break this rule by running a deployment with the `--force` flag.
```
Comment (jdef) [inline docs/docs/deployments.md]: why is `force` prefixed with `--` here? do you have some CLI in mind? Marathon doesn't ship with a CLI out of the box. Pretty sure the original meaning of "flag" here was referring to the use of `force=true` as an API call parameter.

File: docs/docs/deployments.md (hunk 1 above)
Comment (sascala) [inline docs/docs/deployments.md]: What do you mean?

File: docs/docs/deployments.md (hunk 2 above)
Comment (sascala) [inline docs/docs/deployments.md]: My bad. Fixed.

File: docs/docs/deployments.md (hunk 1 above)
Comment (jdef) [inline docs/docs/deployments.md]: "the new deployment request Marathon will reject" ?

Comment (sascala) [inline docs/docs/deployments.md]: indeed. fixed.

**Comment by sascala** [comment]
@jdef Would you mind approving my changes? If, indeed, you approve.

File: docs/docs/deployments.md [hunk 3]
```diff
@@ -5,118 +5,110 @@ title: Application Deployments
...
-A deployment is a set of actions, that can do:
+A deployment is a set of actions, that can:
 
 - Start/Stop one or more applications
 - Upgrade one or more applications
 - Scale one or more applications
 
-Deployments are not instantly available - they take time. 
-A deployment is active in Marathon until the deployment is finished successfully.
+Deployments take time and are not instantly available. A deployment is considered active in Marathon until it has finished successfully.
```
Comment (gkleiman) [inline docs/docs/deployments.md]: Doesn't this give the impression that a deployment is still available, but considered inactive, after it has finished successfully?

Marathon actually removes a deployment once it completes successfully.

File: docs/docs/deployments.md [hunk 4]
```diff
@@ -5,118 +5,110 @@ title: Application Deployments
...
 
-In Marathon there is an upgrade strategy with a minimumHealthCapacity, which is defined on application level.
 The minimumHealthCapacity is a percentage which, when applied to the instance count, defines the number of healthy instances
 that a certain version of the application must have at all times during update.  
 
-- __minimumHealthCapacity == 0__ : all old instances can be killed, before the new version is deployed.
-- __minimumHealthCapacity == 1__ : all instances of the new version is deployed side by side, before the old version is stopped 
-- __minimumHealthCapacity between 0 and 1__ : scale old version to minimumHealthCapacity and start the new version to minimumHealthCapacity side by side. If this is completed successfully then the new version is scaled to 100% and the old version is stopped. 
+- __minimumHealthCapacity == 0__ : All old instances can be killed before the new version is deployed.
+- __minimumHealthCapacity == 1__ : All instances of the new version are deployed side by side before the old version is stopped
```
Comment (gkleiman) [inline docs/docs/deployments.md]: Missing period at the end of the description.

File: docs/docs/deployments.md [hunk 5]
```diff
@@ -5,118 +5,110 @@ title: Application Deployments
...
 that a certain version of the application must have at all times during update.  
 
-- __minimumHealthCapacity == 0__ : all old instances can be killed, before the new version is deployed.
-- __minimumHealthCapacity == 1__ : all instances of the new version is deployed side by side, before the old version is stopped 
-- __minimumHealthCapacity between 0 and 1__ : scale old version to minimumHealthCapacity and start the new version to minimumHealthCapacity side by side. If this is completed successfully then the new version is scaled to 100% and the old version is stopped. 
+- __minimumHealthCapacity == 0__ : All old instances can be killed before the new version is deployed.
+- __minimumHealthCapacity == 1__ : All instances of the new version are deployed side by side before the old version is stopped 
+- __minimumHealthCapacity between 0 and 1__ : Scale the old version to minimumHealthCapacity and start the new version to minimumHealthCapacity side by side. If this is completed successfully, the new version is scaled to 100% and the old version is stopped.
```
Comment (gkleiman) [inline docs/docs/deployments.md]: I'd surround `minimumHealthCapacity` with single quotes.

File: docs/docs/deployments.md [hunk 6]
```diff
@@ -5,118 +5,110 @@ title: Application Deployments
...
-2. Upgrade application app until all instances are replaced, ready and healthy (taking upgradeStrategy into account)  
+1. Upgrade application _db_ until all instances are replaced, ready, and healthy (taking `upgradeStrategy` into account).
+1. Upgrade application _app_ until all instances are replaced, ready, and healthy (taking `upgradeStrategy` into account).
 
-Please take into account that your cluster needs to have more capacity available for the update process if you choose a minimumHealthCapacity greater 0.5.
-In this case more than half of the instances of the same application are run side by side.
-These capacity constraints are summed up if there are dependencies. In our example, we defined 0.6 for db and 0.8 for app. 
-This means that in the update case, we have 12 instances of db (6 old and 6 new) and 32 instances of app (16 old and 16 new) running side by side.
+**Note:** Your cluster needs to have more capacity available for the update process if you choose a `minimumHealthCapacity` greater 0.5. When `minimumHealthCapacity` is greater than 0.5, more than half of the instances of the same application are run side by side.
```
Comment (gkleiman) [inline docs/docs/deployments.md]: Should this be: "if you choose a `minimumHealthCapacity` greater **than** 0.5"?

File: docs/docs/deployments.md [hunk 7]
```diff
@@ -5,118 +5,110 @@ title: Application Deployments
 # Application Deployments
 
 Every change in the definition of applications or groups in Marathon is performed as a deployment.
-A deployment is a set of actions, that can do:
+A deployment is a set of actions, that can:
 
 - Start/Stop one or more applications
```
Comment (gkleiman) [inline docs/docs/deployments.md]: Shouldn't we finish these sentences with periods?

**Comment by aquamatthias** [comment]
@sascala there are some open requests

**Comment by jdef** [comment]
@sascala PTAL

File: docs/docs/deployments.md (hunk 3 above)
Comment (sascala) [inline docs/docs/deployments.md]: added a note

File: docs/docs/deployments.md (hunk 4 above)
Comment (sascala) [inline docs/docs/deployments.md]: 👍

File: docs/docs/deployments.md (hunk 6 above)
Comment (sascala) [inline docs/docs/deployments.md]: yes

**Comment by sascala** [comment]
Thank you for reviewing, @jdef and @gkleiman. I've made the changes.

**Pull request merged**